xforce.json

```diff
@@ -1,16 +1,16 @@
 {
     "appid": "653a4735-1eed-4cb9-98c4-b40408e60b22",
-    "name": "IBM XForce",
-    "description": "This app implements various 'investigative' actions on the 'IBM XForce' device",
+    "name": "IBM X-Force Exchange",
+    "description": "This app implements various 'investigative' actions on the 'IBM X-Force Exchange' device",
     "publisher": "AvantGarde Partners",
-    "package_name": "phantom_xforce",
+    "package_name": "phantom_ibmx-forceexchange",
     "type": "investigative",
-    "license": "Copyright (c) AvantGarde Partners 2017",
+    "license": "Copyright (c) IBM X-Force Exchange",
```

Comment:
What do you think about this?

Reply:
We're changing the "license" field to "copyright" and we need to keep the current copyright

Reply:
Thanks @mattsayar-splunk  for reviewing.
I've made the following updates:
* Renamed `license` to `copyright`
* Kept the `Copyright` value the same